daemon/api_validate.go

```diff
@@ -406,7 +406,9 @@ func getSingleSeqFormingAssertion(st *state.State, accountID, name string, seque
 	}
 
 	sequenceKey := []string{release.Series, accountID, name}
+	st.Unlock()
```

Comment:
we should probably add a pokeStateLock to  func (s *apiValidationSetsSuite) SeqFormingAssertion which should have caught this

Reply:
Added, thanks.